### PR TITLE
MOBILE-2750: Add missing frame customisation params

### PR DESCRIPTION
**Pull request by kacperd**
**What's new**
- Remove code duplication and add `FrameCustomisationHelper` responsible for preparing frame customisation
- Hide language selection 
- Add support for  missing frame customisation parameters: `dbfc` (default button font color), `br` (button radius), `ibc` (input border color)

### CODE REVIEW

File: Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift [hunk 1]
```diff
@@ -78,26 +78,6 @@ internal class KevinPaymentConfirmationViewModel : KevinViewModel<KevinPaymentCo
     }
     
     private func appendUrlParameters(urlString: String) -> URL {
```
Comment (PocketDaniel) [inline Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift:80]: Just wondering, why you left a function here, while all other places you unwrapped that function? There is no benefit to have it here as well, as it just returns `FrameCustomisationHelper` 🤔

File: Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            case defaultButtonFontColor = "dbfc"
+            case buttonRadius = "br"
+            case inputBorderColor = "ibc"
+            case customLayout = "cl"
+
+        }
+    }
+
+    private static var customLayoutElements: [String] {
+        return ["hl"]
```
Comment (PocketDaniel) [inline Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift:72]: What is the reasoning to have it as separate variable?

File: Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            case baseColor = "bsc"
+            case headingsColor = "hc"
+            case fontColor = "fc"
+            case bankIconColor = "bic"
+            case defaultButtonColor = "dbc"
+            case defaultButtonFontColor = "dbfc"
+            case buttonRadius = "br"
+            case inputBorderColor = "ibc"
+            case customLayout = "cl"
+
```
Comment (PocketDaniel) [inline Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift:67]: Nitpick: I think this new line is redundant 😉

**Review by PocketDaniel — APPROVED** [review]

File: Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift (hunk 2 above)
Comment (kacperd) [reply Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift:72]: Initially I planned to put there more content but now it makes no sense :) Will refactor it.

File: Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift (hunk 1 above)
Comment (kacperd) [reply Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift:81]: Changed in a9290544deb2ab6e47f1348aa30a150eccc813cc

File: Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift (hunk 3 above)
Comment (kacperd) [reply Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift]: Changed in a9290544deb2ab6e47f1348aa30a150eccc813cc

File: Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift (hunk 2 above)
Comment (kacperd) [reply Sources/Kevin/Core/Helpers/FrameCustomisationHelper.swift]: a9290544deb2ab6e47f1348aa30a150eccc813cc

**Review by typuk — APPROVED** [review]

**Review by Holospin — APPROVED** [review]

**Pull request merged**